apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/AbstractR4ResourceProvider.java

```diff
@@ -219,13 +232,21 @@ public Bundle findByPatient(
       Bundle bundleResource;
 
       boolean isHashed = !Boolean.FALSE.toString().equalsIgnoreCase(hashed);
+      boolean excludeSamhsa = Boolean.TRUE.toString().equalsIgnoreCase(samhsa);
```

Comment:
Yes, I believe this is currently what the EOB is doing, which is what we were trying to match I think:
https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java#L342-L344

Reply:
Ok, I thought i remembered it matched EOB, but knowing more about samhsa now I was still wondering why we allow it by default. Outside the scope of this PR perhaps 🍰

Reply:
@lsmitchell I believe this can be attributed to the order in which peering APIs went live.  BB was the first API, and it doesn't need any SAMHSA filtering because the benes are allowed to see all the data.  BCDA came online later, and to keep functionality with how BB communicated with BFD, they made the SAMHSA filter default to false.  

I'm sure @karlmdavis has more historical knowledge around the parameter default value, please correct me if any of this is incorrect.